pkg/provisioner/ironic/clients/client.go

```diff
@@ -27,6 +27,8 @@ type TLSConfig struct {
 	ClientPrivateKeyFile  string
 	InsecureSkipVerify    bool
 	SkipClientSANVerify   bool
+	TLSMinVersion         uint16
+	TLSMaxVersion         uint16
```

Comment:
What's the point of this if it's not set in `loadTLSConfigFromEnv()`?

Also the values here are something like 772 for TLS 1.3 (`0x0304` in hex, but that's not much better). That's not a great interface for the user passing these values in the environment.